RELEASE-NOTES.md

```diff
@@ -1,3 +1,53 @@
+#Riak S2 (Riak CS) 2.1.1 Release Notes
+
+## General Information
+This is a bugfix release.
+
+## Bug Fixes
+
+* Remove blocks and a manifest after client failure. If a manifest was
```

Comment:
Network failure is more suitable? Server side NIC failure or router failure also cause the same.